_docs/Contrib.md

```diff
@@ -1,5 +1,86 @@
-* All components should implement the related native HTML component attributes if one exists
-* All input components which are an implementation of an HTML component should have the native input element rendered by them (for SEO and a11y)
+## Accessibility
+
+### Introduction
+
+Implementing accessibility for a component will be done according to the following checklist:
```

Comment:
not clear on first read what's the definition of checking an item in the checklist. makes more sense after reading ` Implementing Accessibility` so maybe the checklist should be a sub-section of the implementation.